fixtures/minuit-une/ivl-photon.json

```diff
@@ -0,0 +1,798 @@
+          "type": "WheelSlot",
+          "comment": "32 Beams"
+        },
+        {
+          "dmxRange": [168, 195],
+          "type": "WheelSlot",
+          "comment": "64 Beams"
+        },
+        {
+          "dmxRange": [196, 223],
+          "type": "WheelSlot",
+          "comment": "128 Beams"
+        },
+        {
+          "dmxRange": [224, 255],
+          "type": "WheelSlot",
+          "comment": "256 Beams"
+        }
+      ]
+    },
+    "Index": {
+      "fineChannelAliases": ["Index fine"],
+      "capability": {
+        "type": "Generic",
+        "comment": "index 360deg"
+      }
+    },
+    "Rotation": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 12],
+          "type": "Rotation",
+          "comment": "Index"
+        },
+        {
+          "dmxRange": [13, 122],
+          "type": "Rotation",
+          "comment": "Rotation fast CCW"
+        },
+        {
+          "dmxRange": [123, 133],
+          "type": "Rotation",
+          "comment": "Rotation stop"
+        },
+        {
+          "dmxRange": [134, 242],
+          "type": "Rotation",
+          "comment": "Rotation fast CW"
+        },
+        {
+          "dmxRange": [243, 255],
+          "type": "Rotation",
+          "comment": "Index"
+        }
+      ]
+    },
+    "Gobo Beam Size": {
+      "fineChannelAliases": ["Gobo Beam Size fine"],
+      "defaultValue": 65535,
+      "highlightValue": 65535,
+      "capability": {
+        "type": "WheelSlot",
+        "comment": "Beam 360deg"
+      }
+    },
+    "Reflector Zoom": {
+      "fineChannelAliases": ["Reflector Zoom fine"],
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "Zoom",
+          "comment": "Mirror"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "Zoom",
+          "comment": "Reflector"
+        }
+      ]
+    },
+    "Reflector Speed": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 0],
+          "type": "Speed",
+          "comment": "Slow"
+        },
+        {
+          "dmxRange": [255, 255],
+          "type": "Speed",
+          "comment": "Fast"
+        }
+      ]
+    },
+    "Tilt 1": {
+      "fineChannelAliases": ["Tilt 1 fine"],
+      "defaultValue": 32767,
+      "highlightValue": 32767,
+      "capabilities": [
+        {
+          "dmxRange": [0, 32767],
+          "type": "Tilt",
+          "angleStart": "0%",
+          "angleEnd": "100%",
+          "comment": "Frost",
+          "menuClick": "center"
+        },
+        {
+          "dmxRange": [0, 65535],
+          "type": "Tilt",
+          "angleStart": "0%",
+          "angleEnd": "100%",
+          "comment": "Tilt Centre",
+          "menuClick": "center"
+        },
+        {
+          "dmxRange": [32767, 65535],
+          "type": "Tilt",
+          "angleStart": "0%",
+          "angleEnd": "100%",
+          "comment": "Reflective",
+          "menuClick": "center"
```

Comment:
I see what you're trying to do here (encode the "frost"/"reflective" sides), but I'm not sure the overlapping dmxRanges will work.  I think we should just do one [0, 65535] range, and put the frost/reflective notes in a comment.